com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/ShaderPass/DotsDeformation.hlsl

```diff
@@ -0,0 +1,61 @@
+#if defined(DOTS_INSTANCING_ON)
+struct DeformedVertexData
+{
+    float3 Position;
+    float3 Normal;
+    float3 Tangent;
+};
+
+uniform StructuredBuffer<DeformedVertexData> _DeformedMeshData;
+uniform StructuredBuffer<DeformedVertexData> _PreviousFrameDeformedMeshData;
+
+void DOTS_Deformation(inout AttributesMesh input)
```

Comment:
The name isn't very descriptive imo.  Perhaps something along the lines of `DOTS_InjectDeformedVertex`?

Reply:
I think maybe we want to put more things in here? I feel like something very general should be used. Maybe even just DotsDeform? Also we probably can move it to core at some point, if/when URP supports motion vectors